app/assets/stylesheets/partials/_admin_workshop.scss

```diff
@@ -2,3 +2,10 @@
   padding-top: 10px;
 }
 
+#workshop_description {
+  height: 150px;
+}
+
+.lead {
```

Comment:
@turger I wonder if there's a Foundation class we can use here instead of defining our own. If not there must be another place we have defined a small class in the CSS, I'm not sure where off the top of my head but I would have a quick look. Or perhaps use the `<small>` tag?

Reply:
Yes! I changed to the small tag and removed this.